Structs/SophonPatchBranch.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Hi3Helper.Sophon.Structs;
+
+#nullable enable
+namespace Hi3Helper.Sophon
+{
+    public partial class SophonPatch
+    {
+        public static async
+            Task<SophonChunkManifestInfoPair>
+            CreateSophonChunkManifestInfoPair(HttpClient client,
+                                              string url,
+                                              string versionUpdateFrom,
+                                              string? matchingField = null,
+                                              CancellationToken token = default)
```

Comment:
CancellationToken.None bc .NET can be .NET

Reply:
CancellationToken.None is not a constant and a struct (except a primitive ones) will **NEVER** be able to get defined as a constant. This won't fix it. You can only define the default value of the parameter if it's a constant and a primitive types.

Read: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1736?f1url=%3FappId%3Droslyn%26k%3Dk(CS1736)

![image](https://github.com/user-attachments/assets/7f469252-4300-48e4-8aa2-a2f95d00bb1c)